packages/lexical/src/LexicalEditorState.ts

```diff
@@ -57,6 +57,9 @@ function exportNodeToJSON<SerializedNode extends SerializedLexicalNode>(
   node: LexicalNode,
 ): SerializedNode {
   const serializedNode = node.exportJSON();
+  if (node.__classes && Object.keys(node.__classes).length > 0) {
```

Comment:
```suggestion
  if (Object.keys(node.__classes).length > 0) {
```

It does seem strange to force this here, before this change `node.exportJSON()` would always export the final result (other than children).